Files/View Models/SettingsViewModel.cs

```diff
@@ -593,10 +593,10 @@ public Type GetLayoutType()
 
         private void DetectGridViewSize()
         {
-            _GridViewSize = Get(125, "GridViewSize"); // Get GridView Size
+            _GridViewSize = Get(Constants.Browser.GridViewBrowser.GRIDVIEW_SIZE_SMALL, "GridViewSize"); // Get GridView Size
         }
 
-        private int _GridViewSize = 125; // Default Size
+        private int _GridViewSize = Constants.Browser.GridViewBrowser.GRIDVIEW_SIZE_SMALL; // Default Size
```

Comment:
Can you change name of field too? `gridViewSize`

Reply:
I have not named it like that but I'll change it